### ISSUE TITLE
Cloud computing

### ISSUE THREAD

**Issue by Darguelles** [perujug/propuestas-meetup#2]
Considero necesario difundir los conocimientos sobre las tecnologías cloud con las que trabajamos así como su correcto uso, existe una gran cantidad de profesionales en Perú que no conocen los distintos tipos de Paas, conceptos de entrega continua, containers, etc.

**Comment by eddumelendez** [comment]
Hola @Darguelles te animas a dar una charla de cloud computing?

**Comment by Darguelles** [comment]
Claro, no me considero un experto pero si conozco algo, si con eso es suficiente cuenta conmigo.

**Comment by Darguelles** [comment]
Man, confirmo mi participación con la de una punta más para cloud y fácil podemos ver algo de microservices tb, pero centrados más en cloud. Me avisas cuando podría ser.

**Comment by eddumelendez** [comment]
Gracias @Darguelles

Podrías indicar lo siguiente:

* Título
* Descripción
* Ponentes

**Comment by Darguelles** [comment]
Listo @eddumelendez 

- 1. Titulo: Cloud in 2017
- 2. Descripción: Vistazo a las herramientas en la nube para potenciar el desarrollo de software y analizar en conjunto la tendencia que seguira este 2017.
- 3. Ponentes: Erickson Murrugarra - Diego Arguelles

Nos confirmas la fecha, gracias por el apoyo. Saludos.
